tests/by-util/test_date.rs

```diff
@@ -409,3 +409,21 @@ fn test_date_overflow() {
         .no_stdout()
         .stderr_contains("invalid date");
 }
+
+#[test]
+fn test_date_parse_from_format() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    const FILE: &str = "file-with-dates";
+
+    std::fs::write(
+        at.plus(FILE),
+        "2023-03-27 08:30:00\n\
+         2023-04-01 12:00:00\n\
+         2023-04-15 18:30:00",
+    )
+    .unwrap();
```

Comment:
```suggestion
    at.write(
        FILE,
        "2023-03-27 08:30:00\n\
         2023-04-01 12:00:00\n\
         2023-04-15 18:30:00"
    );
```

Reply:
Thanks! I commited this via the GH ui as a separate commit but happy to also squash it into my original commit and force push (I don't know (yet) what policies you so I wanted to play it safe :)

Reply:
This is fine, we squash in Github

Reply:
but just don't add lzma files ;)

Reply:
Hahaha - I won't, promised!